plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryConnectorTest.java

```diff
@@ -709,6 +711,14 @@ public void testNativeQuerySimple()
                 "VALUES 1");
     }
 
+    @Test
+    public void testNativeQuerySelectForCaseSensitiveColumnNames()
+    {
+        MaterializedResult result = computeActual
+                ("SELECT * FROM TABLE(bigquery.system.query(query => 'select 1 as lower, 2 as UPPER, 3 as miXED'))");
```

Comment:
Uppercase `select` and `as`. Also, please create a table instead of string literal and assert the rows in addition to the column names.